From 3af743717f94127a037c7e5338b2cb04cd07e1df Mon Sep 17 00:00:00 2001
From: acidicMercury8 <acidicmercury8@outlook.com>
Date: Sun, 16 Mar 2025 14:35:34 +0300
Subject: [PATCH 5/7] Apply fix after revert

---
 src/lj_tab.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/lj_tab.h b/src/lj_tab.h
index 1efa9506..5293f0bd 100644
--- a/src/lj_tab.h
+++ b/src/lj_tab.h
@@ -38,8 +38,8 @@ static LJ_AINLINE Node *hashmask(const GCtab *t, uint32_t hash)
   return &n[hash & t->hmask];
 }
 
-/* String IDs are generated when a string is interned. */
-#define hashstr(t, s)		hashmask(t, (s)->sid)
+/* String hashes are precomputed when they are interned. */
+#define hashstr(t, s)		hashmask(t, (s)->hash)
 
 #define hashlohi(t, lo, hi)	hashmask((t), hashrot((lo), (hi)))
 #define hashnum(t, o)		hashlohi((t), (o)->u32.lo, ((o)->u32.hi << 1))
-- 
2.47.1.windows.2

