From a270d03577c11180396909f838a1c95dcdf4e98c Mon Sep 17 00:00:00 2001
From: acidicMercury8 <acidicmercury8@outlook.com>
Date: Fri, 8 Dec 2023 22:59:11 +0300
Subject: [PATCH 2/2] Delete conflicting functions

---
 src/nvmath/nvmath.h | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/src/nvmath/nvmath.h b/src/nvmath/nvmath.h
index 2a4f887..93a15bf 100644
--- a/src/nvmath/nvmath.h
+++ b/src/nvmath/nvmath.h
@@ -117,18 +117,6 @@ inline float asinf_assert(const float f)
 #define asinf asinf_assert
 #endif
 
-#if NV_CC_MSVC
-NV_FORCEINLINE float log2f(float x)
-{
-    nvCheck(x >= 0);
-    return logf(x) / logf(2.0f);
-}
-NV_FORCEINLINE float exp2f(float x)
-{
-    return powf(2.0f, x);
-}
-#endif
-
 namespace nv
 {
     inline float toRadian(float degree) { return degree * (PI / 180.0f); }
-- 
2.42.0.windows.2

