From 02fc3ef144682c815c94fd1709587ae015dd80c4 Mon Sep 17 00:00:00 2001
From: acidicMercury8 <acidicmercury8@outlook.com>
Date: Wed, 17 May 2023 19:49:52 +0300
Subject: [PATCH 2/2] Fix export definitions

---
 win32/ogg.def | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/win32/ogg.def b/win32/ogg.def
index 030b644..61f5878 100644
--- a/win32/ogg.def
+++ b/win32/ogg.def
@@ -2,7 +2,7 @@
 ; ogg.def
 ; List of exported functions for Windows builds.
 ;
-LIBRARY ogg
+LIBRARY libogg
 EXPORTS
 ;
 oggpack_writeinit
-- 
2.40.0.windows.1

