From d92eb0257776c6e812be2009ac27a2d0e6b7fd53 Mon Sep 17 00:00:00 2001
From: acidicMercury8 <acidicmercury8@outlook.com>
Date: Tue, 9 May 2023 12:32:18 +0300
Subject: [PATCH 4/4] Delete `rint` function from encoder

---
 examples/encoder_example.c | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/examples/encoder_example.c b/examples/encoder_example.c
index 94fc050..8962079 100644
--- a/examples/encoder_example.c
+++ b/examples/encoder_example.c
@@ -51,14 +51,6 @@
 /*supply missing headers and functions to Win32. going to hell, I know*/
 #include <fcntl.h>
 #include <io.h>
-
-static double rint(double x)
-{
-  if (x < 0.0)
-    return (double)(int)(x - 0.5);
-  else
-    return (double)(int)(x + 0.5);
-}
 #endif
 
 const char *optstring = "b:e:o:a:A:v:V:s:S:f:F:ck:d:z:\1\2\3\4";
-- 
2.40.0.windows.1

